include/zephyr/net/dsa_core.h

```diff
@@ -75,6 +75,12 @@ extern "C" {
 	};                                                                                         \
 	DT_INST_FOREACH_CHILD_STATUS_OKAY_VARGS(n, fn, n);
 
+#define DSA_TAG_PROTO_NONE	0
```

Comment:
change it to be:
`#define DSA_TAG_PROTO_NOTAG	0` ?

Reply:
Okay, thanks.